.github/workflows/docker-dev-build.yml

```diff
@@ -15,21 +15,16 @@ on:
 jobs:
   build-front:
     name: Front build
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Why not use `ubuntu-latest`?

Reply:
Using non-fixed version in CI is the best way to have PR broken when Ubuntu publish a big upgrade IMO ^^

I prefer to have stability, and upgrade manually when needed